### PR TITLE
test(e2e,playwright): FourOhFour component

### PR DESCRIPTION
**Pull request by ArjunSingh-Rawat**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

resolves "UI Components/FourOhFour" https://github.com/freeCodeCamp/freeCodeCamp/issues/51705

camperbot cant code

### CODE REVIEW

**Comment by moT01** [comment]
Want to take a look at the failing tests @ArjunSingh-Rawat? It might just need a rebase. There were some issues recently causing them to fail that have been resolved. Let us know if you need any guidance.

**Comment by ArjunSingh-Rawat** [comment]
thanks @moT01 for pointing, can I get help in how should I rebase

File: e2e/four-oh-four.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/404');
+});
+
+test.describe('test for FourOhFour component', () => {
+  test('should display 404 image', async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (ArjunSingh-Rawat) [reply e2e/four-oh-four.spec.ts:9]: @huyenltnguyen I got it, we should write a single test and check all elements of FourOhFour componenet in that.

I have one more doubt, this branch is 68 commits behind should I push merge commit with new changes

**Review by huyenltnguyen — APPROVED** [review]
LGTM.

CI failed on the PR, but it's not your fault 😄 We have a fix ready and are waiting for the checks to complete. We will handle the branch update for you when we are confident with the fix / when `main` is more stable.

Thank you for your contribution!

**Review by raditotev — APPROVED** [review]
LGTM 🚀

**Review by Ahmadkashif — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by ArjunSingh-Rawat** [comment]
thanks @naomi-lgbt 
I am so excited that this PR got merged

**Comment by ArjunSingh-Rawat** [comment]
thanks @naomi-lgbt 
I am so excited that this PR got merged

**Pull request merged**